__tests__/commands/run.js

```diff
@@ -133,3 +144,20 @@ test('adds quotes if args have spaces and quotes', (): Promise<void> => {
     expect(execCommand).toBeCalledWith(...args);
   });
 });
+
+test('adds workspace root node_modules/.bin to path when in a workspace', (): Promise<void> => {
+  return runRunInWorkspacePackage('packages/pkg1', ['env'], {}, 'workspace', (config, reporter): ?Promise<void> => {
+    const logEntry = reporter.getBuffer().find(entry => entry.type === 'log');
+    const logData = logEntry ? logEntry.data.toString() : '{}';
+    const parsed = logEntry ? JSON.parse(logData) : {};
+    let envPaths = [];
+    if (parsed.PATH) {
```

Comment:
Please use [`path.delimiter`](https://nodejs.org/api/path.html#path_path_delimiter) instead. I'll go ahead and fix this myself and then merge tho :)

Reply:
Oh cool, I didn't even know that was a thing. Thanks!